src/Http/PoolFactory.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MagentoCloud\Http;
+
+use GuzzleHttp\Pool;
+use Magento\MagentoCloud\App\ContainerInterface;
+use Magento\MagentoCloud\Util\UrlManager;
+use Psr\Http\Message\RequestInterface;
+
+/**
+ * Create Guzzle request pools.
+ */
+class PoolFactory
+{
+    /** @var ContainerInterface */
+    private $container;
+
+    /** @var ClientFactory */
+    private $clientFactory;
+
+    /** @var RequestFactory */
+    private $requestFactory;
+
+    /** @var UrlManager */
+    private $urlManager;
+
+    public function __construct(
```

Comment:
add phpdoc

Reply:
What is there to document here? Parameter types are already type hinted.

Reply:
You should use PHP doc for all methods, even if it's type-hinted